tests/evm/test_begin_tx.py

```diff
@@ -97,7 +112,18 @@ def test_begin_tx(tx: Transaction, callee: Account, result: bool):
         bytecode_table=set(callee.code.table_assignments(randomness)),
         rw_table=set(
             [
-                (1, RW.Read, RWTableTag.CallContext, 1, CallContextFieldTag.TxId, 0, tx.id, 0, 0, 0),
+                (
```

Comment:
Do you think it's acceptable to skip the format check for test vectors? Like so: https://github.com/appliedzkp/zkevm-specs/blob/45842edfcdeda4744cf5414728c4d32a4a677473/tests/test_state_circuit.py#L27

I think having this exception can make the test code more readable.

Reply:
That sounds good! Will do.